### ISSUE TITLE
🌍 next export - Cannot find module for page: /en/...

### ISSUE THREAD

**Issue by FrancisLeigh** [vinissimus/next-translate#580]
Hello,

I am trying to be able to run `next export` with i18n handling so have taken it to your library to give it a go!

I have set up my project and the built HTML files from `next build` are working fine.
It's when I run `next export` that I am seeing errors.

#### package.json
```json
// scripts
"build": "next build"
"export": "next export"

// dependencies
"next": "^10.1.3",
"next-translate": "^1.0.5",
```

#### i18n.js
```js
module.exports = {
  locales: ['en', 'fr'],
  defaultLocale: 'en',
  pages: {
    '*': ['common'],
  }
}
```

#### next.config.js
```js
const nextTranslate = require('next-translate');

module.exports = nextTranslate({
  future: {
    webpack5: true,
  }
});
```

#### locales/en/common.json
```json
{
  "example": "English yes"
}
```

#### locales/fr/common.json
```json
{
  "example": "French oui"
}
```

#### pages/404.js
```js
import React from 'react';
import useTranslation from 'next-translate/useTranslation';

export default function() {
  const {t, lang} = useTranslation();
  return (
    <div>
      <h1>{t('example')} - Page not found - {lang}</h1>
    </div>
  )
}
```

When I open the `fr/404.html` file generated by `next build` i can see the correct translations e.g
```html
<div>
    <h1>French oui - Page not found - fr</h1>
</div>
```
But when I run `next export` i get the following errors.
```bash
npm run export

> testing-locales-with-next@0.1.0 export
> next export

info  - Using webpack 5. Reason: future.webpack5 option enabled https://nextjs.org/docs/messages/webpack5
info  - using build directory: /Users/francisleigh/Documents/C/testing-locales-with-next/.next
info  - Copying "static build" directory
info  - No "exportPathMap" found in "next.config.js". Generating map from "./pages"
info  - Launching 15 workers
warn  - Statically exporting a Next.js application via `next export` disables API routes.
This command is meant for static-only hosts, and is not necessary to make your application static.
Pages in your application without server-side data dependencies will be automatically statically exported by `next build`, including pages powered by `getStaticProps`.
Learn more: https://nextjs.org/docs/messages/api-routes-static-export
info  - Copying "public" directory
info  - Exporting (4/4)
Error: Cannot find module for page: /en/404
    at pageNotFoundError (/Users/francisleigh/Documents/C/testing-locales-with-next/node_modules/next/dist/next-server/server/require.js:1:384)
    at getPagePath (/Users/francisleigh/Documents/C/testing-locales-with-next/node_modules/next/dist/next-server/server/require.js:1:917)
    at /Users/francisleigh/Documents/C/testing-locales-with-next/node_modules/next/dist/export/index.js:29:237
    at Array.map (<anonymous>)
    at /Users/francisleigh/Documents/C/testing-locales-with-next/node_modules/next/dist/export/index.js:29:101
    at async Span.traceAsyncFn (/Users/francisleigh/Documents/C/testing-locales-with-next/node_modules/next/dist/telemetry/trace/trace.js:5:584) {
  code: 'ENOENT'
}
```

Any help on this would be great! I'm struggling to elegantly have i18n with statically rendered pages 😢 

Thank you

**Comment by aralroca** [comment]
`next export` does not support i18n routing until it is implemented in the Next.js core. More info: https://nextjs.org/docs/advanced-features/i18n-routing#how-does-this-work-with-static-generation

**Comment by FrancisLeigh** [comment]
@aralroca oh i was under the impression I could maybe use this library as a work around.

Given I am not explicitly configuring the `i18n` property of the `next.config.js` shouldn't the `/[locale]/[slug]` routes in the built directory be just handled like nested routes?

**Comment by aralroca** [comment]
`i18n` property in `next.config.js` is added by NextTranslate plugin 😊

Before version 0.19 of next-translate (0.18 for example) we used a workaround to compile all pages under an alternative directory to pages. With the advent of the Next.js i18n routing feature, in version 0.19 we finally removed the workaround to use the Next.js core feature. It has many improvements over what we used before, but the only drawback is that before we did support `next export` and now we will have to wait for Next.js to do it in its core.
